lib/jbuild

```diff
@@ -0,0 +1,8 @@
+(library
+ ((name        conduit)
+  (public_name conduit)
+  (wrapped false)
+  (preprocess (pps (ppx_sexp_conv)))
+  (modules (conduit conduit_trie resolver))
```

Comment:
yeah, I have a rewrite in mind anyway with some of the lessons from httpaf and the new multicore IO work... I suspect that its not worth minor interface breakage, but just head towards a conduit2 that is blazing fast for connection establishment :-)